examples/merkle-tree-cli/README.md

```diff
@@ -0,0 +1,78 @@
+<div align="center">
+
+# Lambdaworks Merklee Tree CLI
+
+Simple Merkle Tree CLI that uses [Poseidon hash](https://www.poseidon-hash.info/).
+
+</div>
+
+### Usage:
+
+**To create a Merkle Tree and save it to a file you can use:**
+
+```bash
+cargo run --release generate-merkle-tree <TREE_PATH>
+```
+
+For example:
+
+```bash
+cargo run --release generate-merkle-tree sample_tree.csv
+```
+
+
+**To generate proof for a Merkle Tree you can use:**
+
+```bash
+cargo run --release generate-merkle-tree <TREE_PATH> <POSITION>
+```
+
+For example:
+
+```bash
+cargo run --release generate-merkle-tree sample_tree.csv 0
+```
+
+**To verify a proof you can use:**
+
+```bash
+cargo run --release generate-merkle-tree <ROOT_PATH> <INDEX> <PROOF_PATH> <LEAF_PATH>
```

Comment:
this should call the `verify` command

Reply:
I could not run this command successfully, I get the error 
```
Error while running command: Custom { kind: Other, error: "InvalidHexString" }
```

It should be explained how to generate the `root` and `leaf` files. 
I tried with a txt file with the plain numbers and I got the earlier error

Reply:
> this should call the `verify-proof` command

Yes, changing it.

> I could not run this command successfully

I've added documentation on the README on how both input files should be.